tests/admin/views/test_releases_v2.py

```diff
@@ -186,6 +203,8 @@ def test_get_releases(api):
                 "read_only": False,
                 "rule_info": {"2": {"product": "Firefox", "channel": "beta"}},
                 "scheduled_changes": [],
+                "product_required_signoffs": {"releng": 1},
+                "required_signoffs": {},
```

Comment:
The penny hasn't dropped on what `product_required_signoffs` is used for in this reporting of release states, where the beta has no required signoffs, nor does a release that isn't mapped to. Something in the UI patches ?

Reply:
Ah, yes. It's exclusive purpose is for knowing whether or not signoff will be required when moving from read-only to read-write on a Release that is not pointed at by a Rule that requires signoff (wow, that's a mouthful...).

Reply:
OK, that makes sense, paired with the signoff check by product (without channel) in `set_read_only`.